oracleserv/sandbox-oracle/oracle/oracle.go

```diff
@@ -105,6 +106,8 @@ func (c *Config) Valid() error {
 
 // Oracle provides services.
 type Oracle struct {
+	srv.UnimplementedSandboxServiceServer
```

Comment:
What does this do?

Reply:
it's a new thing in grpc that will automatically incldue new endpoints that return "Unimplemented" so it's not a compiler error when you add a new endpoint (forwards compatible).

Reply:
It is stuttering a bit, which resulting from other stuttering

```
service SandboxService {
```

Should we rename this to SandboxApp or just Sandbox?

Reply:
Unfortunately `buf format` really wants it to end with "Service"
```
srvpb/v1/oracle.proto:113:9:Service name "Sandbox" should be suffixed with "Service".
```

Reply:
Interesting. Well I guess it's fine if that's the actual recommendation